src/shm_sub.c

```diff
@@ -586,6 +589,7 @@ _sr_shmsub_notify_wait_wr(sr_sub_shm_t *sub_shm, sr_sub_event_t event, uint32_t
  * @param[in] sub_shm Subscription SHM.
  * @param[in] expected_ev  Expected event. Can be:
  *              ::SR_SUB_EV_NONE - just wait until the event is processed, SHM will not be accessed,
+ *              ::SR_SUB_EV_FINISHED - just wait until the event is processed, SHM will not be accessed,
```

Comment:
These descriptions are exactly the same and the difference should be explained somewhere, this seems like the best place.